apps/web/public/static/locales/en/common.json

```
@@ -3302,6 +3308,12 @@
   "error_creating_role": "Error creating role",
   "error_updating_role": "Error updating role",
   "pbac_desc_create_roles": "Create roles",
+  "pbac_desc_manage_roles": "Manage roles",
```

Biome (2.1.2)

[error] The key pbac_desc_manage_roles was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)